how-to/how-to-template-guide.md

```diff
@@ -0,0 +1,149 @@
+- When and why your user might want to perform the task. For example, in a guide for creating a pull request, you might tell users that pull requests are used to let others know about the changes you have pushed to a branch on a repository.
+
+Note: The how-to guide assumes that a user has basic knowledge of the application and knows what they want to achieve.
+
+If there are multiple scenarios under which the task can be completed, then create a table as shown below and hyperlink the different how-to guides available for each scenario.
+
+| Guide                             | Scenario                                             |
+| --------------------------------- | ---------------------------------------------------- |
+| Create a pull request from a fork | The user does not have write permission to the repo. |
+| Create a pull request             | The user has write permission to the repo.           |
+
+## About the "Before you begin" section
+
+In this section describe what your users need to know, or need to have before they attempt the how-to. By stating the requirements up-front, you prevent your users from getting halfway through and discovering that they need to go and read other documentation before they can continue.
+
+Use this section to communicate any prerequisites for this how-to, such as:
+
+- Familiarity with the application
+- Software and tools needed
+- Environments to set up and configure
+- Authentication and authorization information
+- Other guides or information to read
+- Links to procedures or information, or any useful pointers about how to get what they need.
+
+For example:
+
+```markdown
+Before you begin, make sure you meet these prerequisites:
+
+- API credentials for the v3.5 API. For more information about accessing your API credentials, see http://example.com/access_your_api_credentials.
+- Access to the Postman application.
+- A conceptual understanding of RESTful APIs. For more information, see http://example.com/restful_apis.
+- (Optional) A development environment (IDE) that displays API responses formatted for readability.
+- A list of favourites prepared, so you can manage them. For more information about favourites lists, see http://example.com/favorite_lists.
+```
+
+For easy understanding, consider classifying the prerequisites into different categories such as software prerequisites, and hardware prerequisites.
+
+Optionally, provide any helpful cues that signal to a user that they’re probably not in the right place and provide them with more suitable options. For example, If you are a Linux user, then refer to {link to relevant Linux how-to guide}.
+
+## About the "Steps" section
+
+The steps section is where you describe what the user needs to do. Use an ordered list structure to document the how-to steps. How you write your steps will vary depending on your organization's style guide. The template organizes the steps in the following way:
+
+{Task name}
+
```

Comment:
```suggestion

   {Optional: Concise description of the purpose of this section. Only include this if the purpose is not clear from the task title.}

```
suggestion: It would likely be worth adding such information. (Note, template should be updated to reflect this too.)